automation/taskcluster/lib/tasks.py

```diff
@@ -70,14 +70,17 @@ def craft_assemble_release_task(self, architectures, build_type, is_staging, ver
         )
 
         capitalized_build_type = upper_case_first_letter(build_type)
+        coverage_flag = ('-Pcoverage ' if run_coverage else '',)
```

Comment:
Does this make sense to be a tuple? Perhaps `coverage_flag = '-Pcoverage' if run_coverage else ''` would be cleaner?

Reply:
Fixed.